README.rst

```diff
@@ -2,56 +2,127 @@
+
+INH's three types of constraints are:
+
+The "hard shift" constraint requires that at least one nurse is assigned for
+each working day.
+
+The "hard nurse" constraint ensures that no nurse works two or more 
+consecutive days.
+
+The "soft nurse" constraint requires that all nurses should have roughly
+even work schedules.
+
+This demo seeks to obtain reasonable results for a nurse schedule, based on
+INH's model. In our implementation, the number of days is D, and the number 
+of nurses is N. We wish to find a schedule for the N nurses, on the D days, 
+which satisfies the following conditions:
+
+* One, and only one, nurse has been assigned to each day (hard shift 
+  constraint)
+* No nurse works two days in a row (hard nurse constraint)
+* The nurses should work the same number of days
+
+Running the demo results in the following output, at the command-line:
+
+    Size  33
+
+    Energy  0.5999999999999694
+
+    Checking Hard shift constraint  0.0
+
+    Checking Hard nurse constraint  0.0
+
+    Checking Soft nurse constraint  0.6
+
+========= =  =  =  =  =  =  =  =  =  =  ==
+   Day    0  1  2  3  4  5  6  7  8  9  10
+========= =  =  =  =  =  =  =  =  =  =  ==
+Nurse  0           X     X     X     X   
+Nurse  1     X        X     X           X
+Nurse  2  X     X                 X      
+========= =  =  =  =  =  =  =  =  =  =  ==
+
+The results show the following:
+
+* One, and only one, nurse has been assigned to each day
+* No nurse works two days in a row
+* Two nurses work 4 days, and one works three days
 
 Usage
 -----
 
-A simple command that runs the demo. For example,
+To run the demo, run the command
 
 .. code-block:: bash
 
-  python <demo_name>.py
+  python nurse_scheduling.py
 
 
 Code Overview
 -------------
 
-A general overview of how the code works.
+Here is a general overview of the Nurse Scheduling code:
 
-Prefer bite-sized descriptions in bullet points:
-
-* Here's an example bullet point
+* Assign the size of the problem (number of nurses and days) and parameters
+* Compute the "penalty matrix" J
+* Develop the QUBO matrix
+* Run the problem (solve the QUBO)
+* Compute the hard shift constraint sum
+* Compute the hard nurse constraint sum
+* Compute the soft nurse constraint sum
+* Print the nurse schedule
 
+Note that the total of the three constraint sums should equal the energy.
 
 Code Specifics
 --------------
 
-Notable parts of the code implementation.
+Some notes on the code:
 
-This is the place to:
+* We use a vector to store the two-dimensional (nurse, day) matrix:
 
-* Highlight a part of the code implementation
-* Talk about unusual or potentially difficult parts of the code
-* Explain a code decision
+(0, 0) (0, 1) (0, 2)... (0, D) (1, 0) (1, 1)... (1, D)
 
-Note: there is no need to repeat everything that is already well-documented in
-the code.
+* The three constraint sums are separated out in order to be able to 
+confirm the individual effects manually. For example, if a nurse was
```

Comment:
If you want to make this whole sentence as a bullet you need to space the three lines below "The" in the first line:
![image](https://user-images.githubusercontent.com/34041130/79247774-ca405d80-7e2f-11ea-8555-847049c8dc53.png)

Reply:
Got it, thanks.